Formula/p/python-pkgconfig.rb

```diff
@@ -0,0 +1,36 @@
+class PythonPkgconfig < Formula
+  include Language::Python::Virtualenv
+
+  desc "Interface Python with pkg-config"
+  homepage "https://github.com/matze/pkgconfig"
+  url "https://files.pythonhosted.org/packages/c4/e0/e05fee8b5425db6f83237128742e7e5ef26219b687ab8f0d41ed0422125e/pkgconfig-1.5.5.tar.gz"
+  sha256 "deb4163ef11f75b520d822d9505c1f462761b4309b1bb713d08689759ea8b899"
+  license "MIT"
+
+  depends_on "poetry" => :build
+  depends_on "python@3.12" => [:build, :test]
```

Comment:
```suggestion
  depends_on "python@3.11" => [:build, :test]
  depends_on "python@3.12" => [:build, :test]
```

Reply:
This cannot be done, as `poetry`  depends only on `python@3.12`.

Reply:
I dont think that is the case, I have actually used poetry for python-rich formula build. (poetry depends on py3.12, but it is just build dependency in here)